.github/workflows/pythonapp.yml

```diff
@@ -30,8 +30,11 @@ jobs:
           # stop the build if there are Python style violations
           flake8 . --count --show-source --statistics
           flake8 tests --config tests/.flake8 --count --show-source --statistics
-      - name: Generate test parser
-        run: java -cp ../antlr-4.10.1-complete.jar org.antlr.v4.Tool -Dlanguage=Python3 -visitor -o simple_lang SimpleLangLexer.g4 SimpleLangParser.g4
+      - name: Generate test parsers
```

Comment:
Would it be useful to put these into a script, so that it can be used also when running tests locally?

Reply:
Yes, it makes sense